src/lib/core/ripple/ripple.ts

```diff
@@ -100,7 +100,7 @@ export class MatRipple implements OnChanges, OnDestroy {
   }
 
   ngOnChanges(changes: SimpleChanges) {
-    if ((changes['trigger'] || changes['_matRippleTrigger']) && this.trigger) {
+    if (changes.trigger && this.trigger) {
```

Comment:
`trigger` needs to stay as a string or else Closure Compiler might rename it (which we don't want for input names)

Reply:
Makes sense. Changed it back to be on the safe side.